RedHat-UX/red-hat-design-system
# docs(alert): migrate docs to /elements

Closes #820

## What I did

1. 50c76b59 docs: use markdown-it-attrs
    - lets us inline styles, classes, ids, etc in markdown
1. ff62ded4 refactor(docs)!: make example shortcode an image
    - simplifies using the example docs component (todo: webc it)
1. ecc892ea docs(footer): move installation instructions to code page
3. 7cdbcdb9 docs(alert): migrate docs to /elements
4. 7170b778 docs: copy elements docs assets
5. 9f671c5b docs: unify feedback partial for footer, alert
6. 391d298d docs(footer): move assets to /elements
7. 922f16a7 docs(alert): move assets to /elements



## Testing Instructions

1. check the [dialog docs](https://deploy-preview-819--red-hat-design-system.netlify.app/components/dialog/) for visual regressions
2. check the cheatsheet for visual regressions (or skip, since it's dev-only)
3. check the footer [overview](https://deploy-preview-819--red-hat-design-system.netlify.app/components/footer/) and [code](https://deploy-preview-819--red-hat-design-system.netlify.app/components/footer/code/) tabs for sensibility
5. check the [new alert pages](https://deploy-preview-819--red-hat-design-system.netlify.app/components/alert) for regressions

DP demos:

- [alert](https://deploy-preview-819--red-hat-design-system.netlify.app/elements/alert/demo/) 
- [footer](https://deploy-preview-819--red-hat-design-system.netlify.app/elements/footer/demo/) 
- [footer universal](https://deploy-preview-819--red-hat-design-system.netlify.app/elements/footer/demo/footer-universal/)

## Notes to Reviewers

- Check for the correctness of content structure
- check the correctness of heading levels / presence of interleaving content
If enough *content* changes are likely that it will hold up merging past EoW, please make a new issue for those changes, as this PR has infra stuff in it. Alternately, we can split this into two PRs.
